spec/helpers/stylesheet.js

```diff
@@ -25,7 +25,7 @@ describe('stylesheet helper', () => {
     it('should render a link tag and all extra attributes with no cdn url', done => {
         runTestCases([
             {
-                input: '{{{stylesheet "assets/css/style.css" blah rel="something" class="myStyle"}}}',
+                input: '{{{stylesheet "assets/css/style.css" rel="something" class="myStyle"}}}',
```

Comment:
Why did this test need to change?

Reply:
test needed to change as more than one argument is being passed to stylesheet.js helper.  We do not have access to options in helper if that is the case.